.circleci/config.yml

```diff
@@ -93,6 +93,7 @@ jobs:
       IS_LOCAL_DEVELOPMENT: false
       OK_TO_RELEASE_CORE: true
       OK_TO_RELEASE_TELEM: true
+      OK_TO_RELEASE_LITE: false
```

Comment:
why not just release both at the same time? any potential issues you see?

Reply:
work is still in progress. Working on being able to release separately and then will be easy enough to combine if necessary.